beta/src/pages/learn/updating-objects-in-state.md

```diff
@@ -1,57 +1,57 @@
 ---
-title: Updating Objects in State
+title: स्टेट में ऑब्जेक्ट्स को अपडेट करना
 ---
 
 <Intro>
 
-State can hold any kind of JavaScript value, including objects. But you shouldn't change objects that you hold in the React state directly. Instead, when you want to update an object, you need to create a new one (or make a copy of an existing one), and then set the state to use that copy.
+स्टेट ऑब्जेक्ट समेत कोई भी जावास्क्रिप्ट वैल्यू स्टोर कर सकती है। लेकिन आपको कभी भी  ऑब्जेक्ट्स को बदलना नहीं चाहिए जो आप React स्टेट में रखते हैं। उसके बजाए आपको जब भी  ऑब्जेक्ट को अपडेट करना हो या तो आप  एक नया ऑब्जेक्ट बनाये  (या उसी की एक कॉपी बनाये) और फिर उस कॉपी का उपयोग करने के लिये स्टेट को सेट कर दें। 
 
 </Intro>
 
 <YouWillLearn>
 
-- How to correctly update an object in React state
-- How to update a nested object without mutating it
-- What immutability is, and how not to break it
-- How to make object copying less repetitive with Immer
+- React में सही से ऑब्जेक्ट को कैसे अपडेट करे
+- बिना बदले कैसे नेस्टेड ऑब्जेक्ट को अपडेट करे 
+- इम्म्यूटेबलिटी  क्या होती है और कैसे उसे ब्रेक ना करें 
+- Immer की मदद से बार-बार ऑब्जेक्टस की कॉपी बनाने को कम कैसे करें
 
 </YouWillLearn>
 
-## What's a mutation? {/*whats-a-mutation*/}
+## म्युटेशन क्या होती है ?  {/*whats-a-mutation*/}
```

Comment:
`होती है ?` => `होता है?`

Reply:
@Akshat2Jain There is a typo in this `होता है? ?` you have added two question marks